lib/utils.js

```diff
@@ -86,7 +86,7 @@ export function useNow(updateEvery = 1000) {
 }
 
 export function usePostEmail() {
-  const { account } = useWeb3Connect() || ''
+  const { account } = useWallet() || ''
```

Comment:
I didn't want to run the event logging code in tests so grabbed the account directly.

Reply:
Ah, the analytics code should be set up to skipped during non `NODE_ENV=production` environments; I might've forgot to do this when initially setting up the analytics 🤭